### PR TITLE
Update Windows manifest

### PR DESCRIPTION
**Pull request by n0toose**
- Updated tagline
- Removed reference to Audacity public key

<details>
<summary>Checklist</summary>

- [x] I have signed off my commits using `-s` or `Signed-off-by`\* (See: [Contributing § DCO](https://github.com/tenacityteam/tenacity/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code\*
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving\*
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"\*

\* indicates required

</details>

### CODE REVIEW

File: win/Tenacity.exe.manifest [hunk 1]
```diff
@@ -21,7 +21,7 @@
   </trustInfo>
   <dependency>
     <dependentAssembly>
-      <assemblyIdentity type='win32' name='Microsoft.Windows.Common-Controls' version='6.0.0.0' processorArchitecture='*' publicKeyToken='6595b64144ccf1df' language='*' />
```
Comment (nbsp) [inline win/Tenacity.exe.manifest:24]: What is the public key for? If it's a PGP signing key, we could substitute it for our own.

Comment (n0toose) [reply win/Tenacity.exe.manifest:24]: It's for the Windows certificate. We cannot substitute it yet.

**Review by nbsp — CHANGES_REQUESTED** [review]
s/Updated/Update

**Comment by n0toose** [comment]
Ouch, true that.

**Review by nbsp — APPROVED** [review]

**Comment by n0toose** [comment]
Merged in 20ee9a6.

**Comment by emabrey** [comment]
This broke the executable manifest badly enough that I had to use `sxstrace` to figure out what the problem was because I couldn't even attach a regular debugger. `6595b64144ccf1df` has nothing to do with Audacity; it is actually the VS6+ common control artifact version hash and the name is a bit misleading unless you do a bit of searching. We need to be testing that the application opens if we make changes to the EXE manifest. Additionally, if we are making platform specific changes, we need to let someone who is running that platform verify it for us if we don't personally have access to the OS machines.

Also, please add `Reference-to: <link to PR>` to your merge commits. It makes it much less clear exactly what PR is related to the merge otherwise.

I've reverted this change in master after confirming it fixes the Windows executable.

**Comment by n0toose** [comment]
At least I learned something new today at the expense of others suffering, yay!

Anyways, for the record, the change was reverted here: https://github.com/tenacityteam/tenacity/commit/4e877b8aed62d3318212e1a863b1c0212787bede

**Pull request closed without merging**